frontend/src/cljs/imas_seamap/events.cljs

```diff
@@ -140,7 +141,7 @@
                (merge {:habitat-colours habitat-colours
                        :habitat-titles  habitat-titles
                        :sorting         sorting}))]
-    (assoc-in db [:map :active-layers] (vec (applicable-layers db :category :habitat)))))
+    (assoc-in db [:map :active-layers] (or (get-in db [:map :keyed-layers :startup]) []))))
```

Comment:
`get-in` has a default-value option, no need for `or`

Reply:
Ah yep, thanks!